### PR TITLE
[17.0][IMP] base_import_pdf_by_template: Add dd/MM/YY date_format compatibility

### PR DESCRIPTION
**Pull request by victoralmau**
Changes done:
- [x] Add dd/MM/YY `date_format` compatibility
- [x] Add dd/MM/yy `date_format` compatibility
- [x] Add B d, YY `date_format` compatibility 
- [x] Add file_name to view to display the file correctly
- [x]  Improve the display of line data in the form view
- [x] Add message_log in the correct format (html)

Please @CarlosRoca13 and @pilarvargas-tecnativa can you review it?

@Tecnativa TT50003

### CODE REVIEW

**Review by pilarvargas-tecnativa — APPROVED** [review]
LGTM, thanks!

File: base_import_pdf_by_template/views/base_import_pdf_template_line_views.xml [hunk 1]
```diff
@@ -19,68 +19,69 @@
             <form>
                 <sheet>
                     <group name="main_group">
-                        <group name="left">
```
Comment (pedrobaeza) [inline base_import_pdf_by_template/views/base_import_pdf_template_line_views.xml:22]: Why removing groups? At first sight, I prefer to have it into 2 columns.

File: base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py [hunk 2]
```diff
@@ -143,28 +145,30 @@ def _process_set_value_form(self, _form, field_name, value):
...
-                    %(old_value)s</p>"""
+                text = _(
+                    """%(item_name)s has been set with %(new_value)s instead of
+                    %(old_value)s"""
                 ) % {
                     "item_name": getattr(_form, "name"),  # noqa: B009
                     "old_value": old_value_data,
                     "new_value": new_value_data,
                 }
+                if not self.log_text:
```
Comment (pedrobaeza) [inline base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py]: Don't repeat this all the times. Extract it to a method, and don't repeat also the Markup creation.

**Review by pedrobaeza — COMMENTED** [review]
I think you can squash all the commits into one. Just enumerate all the changes in the expanded commit message.

File: base_import_pdf_by_template/views/base_import_pdf_template_line_views.xml (hunk 1 above)
Comment (victoralmau) [reply base_import_pdf_by_template/views/base_import_pdf_template_line_views.xml:22]: I think an example may help to understand it

**Before**
antes

**After**
despues

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**